### PR TITLE
show current number of nodes

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/pull/2206. Note that this PR is based on [`feature-auto-scaling`](https://github.com/giantswarm/happa/pull/424). This is pretty much WIP as of now since I am rather lost in the react magic. Marian wanted to check it out by chance. Let's see.

### CODE REVIEW

**Comment by marians** [comment]
@tuommaki Feel free to delete the taylorbot comments if you find them disturbing. These have been introduced by a PR I merged today.

**Comment by tuommaki** [comment]
CI Green. Now this should show the current desired & present number of nodes correctly.

@giantswarm/sig-ux PTAL :)

File: src/reducers/appReducer.js [hunk 1]
```diff
@@ -49,6 +49,9 @@ export default function appReducer(
         provider: '',
       },
     },
+    status: {
```
Comment (tuommaki) [inline src/reducers/appReducer.js]: Hmm... This is probably not needed? Or should I do something properly here and change this?

File: src/reducers/clusterReducer.js [hunk 2]
```diff
@@ -179,6 +179,24 @@ export default function clusterReducer(
...
+
+      return {
+        lastUpdated: state.lastUpdated,
+        isFetching: false,
+        items: items,
+      };
+
+    case types.CLUSTER_LOAD_STATUS_ERROR:
+      return {
+        numberOfNodes: 0,
```
Comment (tuommaki) [inline src/reducers/clusterReducer.js]: Should this be something different? I don't think my code per se uses this, so what would be correct here?

Comment (pipo02mix) [reply src/reducers/clusterReducer.js]:
```suggestion

        lastUpdated: state.lastUpdated,
        isFetching: false,
        items: items,
```

Indeed I would return the state without changes. Same as we do with other errors

File: src/reducers/appReducer.js (hunk 1 above)
Comment (pipo02mix) [reply src/reducers/appReducer.js]: I dont think it is needed. AFAIK `status` belongs to cluster state

**Comment by xh3b4sd** [comment]
Do you want me to test that locally? CI is red.

File: src/reducers/clusterReducer.js (hunk 2 above)
Comment (tuommaki) [reply src/reducers/clusterReducer.js]: Roger. I checked example from load cluster details error and it's same here now.

File: src/reducers/appReducer.js (hunk 1 above)
Comment (tuommaki) [reply src/reducers/appReducer.js]: ACK. Removed.

**Comment by oponder** [comment]
BTW I made a PR to add the DCO and License here: https://github.com/giantswarm/happa/pull/445

**Pull request merged**